pkg/registry/search/storage/cache.go

```diff
@@ -40,7 +40,7 @@ func (r *SearchREST) newCacheHandler(info *genericrequest.RequestInfo, responder
 
 	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
 		enc := json.NewEncoder(rw)
-
+		rw.Header().Add("Content-Type", "application/json")
```

Comment:
I think `rw.Header().Set("Content-Type", "application/json")` is better.
If with Add, there could be multiple `application/json;application/json;application/json`